docker/bench/bench.Dockerfile

```diff
@@ -39,6 +35,10 @@ EXPOSE 9101
 # Define MINT_KEY, AC_HOST and AC_DEBUG environment variables when running
 ENTRYPOINT ["/opt/libra/bin/bench_init.sh"]
 
+ARG BUILD_DATE
+ARG GIT_REV
```

Comment:
If we’re letting the the build script pull the git rev then we don’t need this envvar anymore. Also, what is the upstream envvar used for?

Reply:
IMO we still need them to tag the image.  Dockerfile syntax doesn't support setting env var or label dynamically.  We can do something like "CMD export GIT_REV=$(git blah)", but that env var only exists in the current layer.  The LABEL step won't see this var in its layer. We can also hack around it by piping the git rev to a file, but it would cause cache miss for subsequent builds (ex validator).

Reply:
> what is the upstream envvar used for?
It is used in the `vcs-upstream` tag, but I don't know who consumes it.  AFAIK, it's not a reserved tag.  Probably for the operators to tell personal docker images apart from the nightly in the ECR.

Reply:
Putting down a TODO note: find out if we can pipe the git rev and versioning info needed for making a release into a file so that we can look it up here.